src/constants/tokens.ts

```diff
@@ -100,6 +103,12 @@ export const RealWorldAssetIndex: Token = {
   symbol: 'RWA',
 }
 
+export const USDCY: Token = {
+  ...getIndexTokenData('USDCY', 1)!,
+  // FIXME: delete; just for testing
+  address: '0xc30FBa978743a43E736fc32FBeEd364b8A2039cD',
```

Comment:
just for testing (MMI): https://etherscan.io/address/0xc30fba978743a43e736fc32fbeed364b8a2039cd

they both use the same issuance: https://etherscan.io/address/0xa0a98eb7af028be00d04e46e1316808a62a8fd59#readContract